src/integrationtests/java/com/aws/greengrass/integrationtests/deployment/DeploymentTaskIntegrationTest.java

```diff
@@ -818,7 +818,7 @@ void GIVEN_services_running_WHEN_service_added_and_deleted_THEN_add_remove_servi
         groupToRootComponentsTopics.lookupTopics("CustomerApp")
                 .replaceAndWait(ImmutableMap.of(GROUP_TO_ROOT_COMPONENTS_VERSION_KEY, "1.0.0"));
         groupToRootComponentsTopics.lookupTopics("YellowSignal")
-                .replaceAndWait(ImmutableMap.of(GROUP_TO_ROOT_COMPONENTS_VERSION_KEY, "1.0.0"));
+                .replaceAndWait(ImmutableMap.of(GROUP_TO_ROOT_COMPONENTS_VERSION_KEY, ">=1.0.0"));
```

Comment:
Is this replacing a test? Looks like we'll now check that it works with a version range, but no longer check that it works with an exact version. Should we add a check to have both?

Reply:
No, there are plenty of tests to cover the previous situation, including the line above this

Reply:
Sounds good.